examples/differential_evolution_4Jcell.py

```diff
@@ -0,0 +1,484 @@
+# try optimizing an anti-reflection coating
```

Comment:
This example, more than an example, is a full-featured tutorial. It's absolutely fine - and increadibly useful as a real-world application -, but it would be useful to have some sort of minimum working example showing how the optimizar is used. Something simple of the type "Adjust the bandgap of a 1J solar cell for best performance under the AM1.5g spectrum" or something like that that can be done in a handful of lines of code.